test/deny_SUITE.erl

```diff
@@ -68,27 +72,7 @@ test_deny(_Config) ->
 
     Options = #{},
 
-    Json = <<"{
-\"action\": \"deny\",
-\"target\": {
-    \"type\": \"cybox:Network_Connection\",
-    \"specifiers\": {
-        \"Layer4Protocol\": \"UDP\",
-        \"DestinationSocketAddress\": {
-            \"IP_Address\": {\"Address_Value\": \"1.2.3.4\"},
-            \"Port\": {\"Port_Value\": 443}
-            }
-        }
-    },
-\"actuator\": {
-    \"type\": \"network-router\",
-    \"specifiers\": {\"port\": \"2\"}
-    },
-\"modifiers\": {
-    \"response\": \"ack\",
-    \"where\": \"perimeter\"
-    }
-}">>,
+    Json = ?JSON_DENY_01,
```

Comment:
The variable "JSON_DENY_01" on line 75 does not respect the format defined by the regular expression '"^([A-Z][0-9a-zA-Z]*)$"'.